code/game/objects/items/trader.dm

```diff
@@ -1049,4 +1066,5 @@
 /obj/item/weapon/storage/cargocache/distance_interact(mob/user)
 	if(istype(loc,/obj/structure/fakecargoposter) && user.Adjacent(loc))
 		return TRUE
-	return FALSE
\ No newline at end of file
+	return FALSE
+>>>>>>> 911d74db74ee85cd238211819d4dfb75037329a1
```

Comment:
based and rebasepilled (also note <<< at 992)

Reply:
ssh